instrumented/nightly-tests/src/androidTest/kotlin/com/datadog/android/nightly/rum/RumMonitorBackgroundE2ETests.kt

```diff
@@ -103,12 +104,13 @@ class RumMonitorBackgroundE2ETests {
     fun rum_rummonitor_add_background_non_custom_action_with_no_outcome() {
         val testMethodName = "rum_rummonitor_add_background_non_custom_action_with_no_outcome"
         val actionName = forge.anActionName()
+        val actionType = forge.aValueFrom(
+            RumActionType::class.java,
+            exclude = listOf(RumActionType.CUSTOM)
+        )
         measure(testMethodName) {
             GlobalRum.get().addUserAction(
-                forge.aValueFrom(
-                    RumActionType::class.java,
-                    exclude = listOf(RumActionType.CUSTOM)
-                ),
+                actionType,
                 actionName,
                 attributes = defaultTestAttributes(testMethodName)
```

Comment:
probably we should also extract `attributes` here as it was done in other places

Reply:
Good catch